### PR TITLE
Update to ES6, expose getNativeBinary

### PR DESCRIPTION
**Pull request by vicb**
@alexeagle

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
 'use strict';
 
-var fs = require('fs');
-var os = require('os');
-var path = require('path');
-var resolve = require('resolve').sync;
-var spawn = require('child_process').spawn;
-var glob = require('glob');
-var async = require('async');
+const fs = require('fs');
```
Comment (alexeagle) [inline index.js:4]: you changed this file to no longer be executable, is that intended?

**Review by alexeagle — APPROVED** [review]

**Comment by vicb** [comment]
No. I am not sure how I did that. I'll update.

File: index.js (hunk 1 above)
Comment (vicb) [reply index.js:4]: fixed. PTAL.

**Comment by mprobst** [comment]
@vicb can you change the commit description to include *why* we expose the native binary? sounds like that should be an implementation detail.

**Comment by vicb** [comment]
Rationale is I was trying to write a clang-format plugin for [precise-commits](https://github.com/nrwl/precise-commits) and needed something synchronous. Up for debate.

**Review by mprobst — APPROVED** [review]

**Pull request merged**